readme.md

```diff
@@ -1,292 +1,165 @@
-  var myClass = "WDI7";
+**Brief history:** Created in 10 days by Brendan Eich, of Mozilla. *Not* related to Java in any way but its name.
+* "Java" is to "Javascript" as "ham" is to "hamster"
 
-  // After instantiation you can then reference variables by just their name, without "var".
-  myClass;
-  => "WDI7"
+**Q:** What's a programming language?  
+* What can it do that a markup language like HTML can't?
+* It let's us do things! It lets us act on information, manipulate it, display it, pretty much whatever we want.
 
-  // Variables can not only store single data types but also expressions.
-  var multiplication = 5 * 2;
-  => 10
-  ```
+Javascript enables us to do all that in a browser.  
+* Using the tools you learned in the pre-work (e.g., data types, loops, functions).
 
-- Javascript is a "dynamic" or "untyped" language, meaning a variable can switch between data types.
+### Why is it the dominant programming language of the web?
 
-  ```javascript
-  // This change is valid in Javascript.
-  var myFavoriteNumber = 5;
-  var myFavoriteNumber = "five";
-  ```
+Barriers to entry for learning Javascript are very low.
+* No additional software required to run it. Just a text editor and a browser.
+* You can even run it directly in the browser via its Javascript console.
+  * Ex. Hide images on the GA website.
+* On top of that, it's supported by all web browsers.
 
-## Numbers (10/45)
+Javascript has evolved since its creation.
+* One of the biggest additions to JS was AJAX, which allows use to reload parts of a page without refreshing the entire thing (just like on Facebook). Big implications for User Experience.
 
-In Javascript, numbers are numerical values -- straightforward!
-  - All numbers are of type "number," regardless of format (e.g., integer, float/decimal).
+A lot of frameworks and libraries -- like Backbone and jQuery -- have emerged that enable us to do so much more -- and do it quickly -- with Javascript.
 
-  ```javascript
-  // The "typeof" operator returns the data type of a value
-  typeof 5;
-  => "number"
+# Setting up our environment (5 minutes, 0:30)
 
-  typeof 5.5;
-  => "number"
-  ```
+## First, create your HTML and JS
 
-Operations
-- Math in Javascript follows the same rules you've known since elementary school math.
-- Basic operators: `+, -, *, /`
+```bash
+# In Terminal...
+$ touch index.html script.js
+```
 
-  ```javascript
-  // Addition
-  10 + 2;
-  => 12
+```html
+<!-- index.html -->
 
-  // Subtraction
-  10 - 2;
-  => 8
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Yo</title>
+    <script src="script.js"></script>
+  </head>
+  <body>
 
-  // Multiplication
-  10 * 2;
-  => 20
+  </body>
+</html>
+```
 
-  // Division
-  10 / 2;
-  => 5
-  ```
+## Next, open the site in Chrome and open the Dev Tools
 
-- Order of precedence: P(E)MDAS (Parentheses, Multiplication, Division, Addition, Subtraction)
+Open the Javascript console in the browser using the following shortcut: `Command + Option + I`
```

Comment:
I is the element inspector, type esc key to toggle. Or cmd option j to go straight to console